src/dispatch/case/models.py

```diff
@@ -106,6 +110,16 @@ class Case(Base, TimeStampMixin, ProjectMixin):
         "Group", backref="case", cascade="all, delete-orphan", foreign_keys=[Group.case_id]
     )
 
+    participants = relationship(
+        Participant,
+        backref="case",
+        cascade="all, delete-orphan",
+        foreign_keys=[Participant.case_id],
+    )
+
+    observer_id = Column(Integer, ForeignKey("participant.id"))
+    observer = relationship(Participant, foreign_keys=[observer_id], post_update=True)
```

Comment:
Do we need an observer for cases? I'm not sure how you determine if someone was an observer. 

For incidents, we define an observer as someone in the channel who doesn't say anything. Since cases will be threaded messages, you could view the thread and not say anything... but I don't think Dispatch would be able to tell that is the case.

Reply:
If we determine that we need case participants with an observer (aka subscriber or follower) role we should leverage the observer participant role we already have available. Perhaps observers can be added via engagement filters and managed using a Slack command? There might be people who are interested in keeping an eye on specific case types.

Reply:
I'm thinking about scrapping the observer role for this PR and just getting participants into the model. Can add observer or similar in a follow up PR. It's also unclear to me how observer would work from a Slack perspective when these are in threads.